.github/workflows/ci.yaml

```diff
@@ -45,12 +43,38 @@ jobs:
           cache: poetry
       - name: Install package
         run: poetry install -E crypto
+  
+  test:
+    needs: lint
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python: ["3.8", "3.9", "3.10", "3.11"]
+    steps:
+      - uses: actions/checkout@v3
+      - run: pipx install poetry
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python }}
+          cache: poetry
+      - name: Install package
+        run: poetry install -E crypto
+      - name: Start integration services
+        run: |
+          timeout 120s ./scripts/start-docker.sh
```

Comment:
I don't know, really. I just copied this stuff from nio-pipelines, where its `start-docker.sh` script has a little more code in it than ours. I'd prefer to just leave the script in here since it might be familiar to some.